build-android/vulkan-validationlayers_revision_android

```diff
@@ -1 +1 @@
-main
+c935d1e214006b40743647572144c2ce3882e4aa
```

Comment:
Is there something special about this commit, or could this point to the v1.3.247 tag?

Reply:
I picked this at semi random. I didn’t go for the 247 tag because I feared that it would need newer headers but that may be a non issue. Then again having 246 and 247 tags is mighty confusing.